gigaword-indexer/src/main/java/edu/isi/vista/gigawordIndexer/Article.java

```diff
@@ -0,0 +1,41 @@
+package edu.isi.vista.gigawordIndexer;
+
+public class Article {
+
+  private String id;
+  private String text;
+  private int segments = 0; // LTF only
```

Comment:
Avoid default values - use `OptionalInt`

Reply:
Renamed

Reply:
I don't see the rename?